GG/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ endif()
 ## Collect project dependencies.
 ##
 
-set(MINIMUM_BOOST_VERSION 1.60.0)
+set(MINIMUM_BOOST_VERSION 1.69.0)
```

Comment:
Whi minimum is not 1.71.0?

Reply:
We decided on that version: https://www.freeorion.org/forum/viewtopic.php?p=111046#p111046

It allows to support Fedora 32 distributive. Do you need something from 1.71?

Reply:
Ubuntu 18.04 is the last Ubuntu which supports 32bit platform and it is supported until 26. 4. 2023. After then, only Debian(if hw firmware and soft possible) and although for 32bit platform is maybe time for upgrade, maybe there is some ppa or to compile boost for 18.04 and decision should be for user to decide, but at least there should be some documentation or to use older release...

Reply:
We are going to raise python version too so Ubuntu 18.04 still will be dropped.

Reply:
Debian 11 is stable so I think it's fine to switch to it for x86.

Reply:
According https://github.com/actions/virtual-environments from [Github docs](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-software) there is no support for Debian, only by [Docker image](https://hub.docker.com/_/debian?tab=tags).

BTW, https://packages.debian.org could help, but maybe as new pull request(currently is FO on Windows on amd64_x86 arch and things theoretically could broke on x86 architecture unnoticed(as on some other architecture...))